kuma/users/jinja2/account/logout.html

```diff
@@ -8,12 +8,11 @@
   <h1>{{ _("Sign Out") }}</h1>
 
   <p>{{ _("Are you sure you want to sign out?") }}</p>
-
-  <form method="post" action="{{ url('account_logout') }}">
+  <form method="post" action="{{ url('allauth_account_logout') }}">
     {% csrf_token %}
-    {% if redirect_field_value %}
-    <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}"/>
-    {% endif %}
+      {% if redirect_value %}
```

Comment:
``redirect_field_value`` is the correct [context variable name](https://github.com/pennersr/django-allauth/blob/0.24.1/allauth/account/views.py#L665)